File: WebClient SDK/src/default/workflow/krProcess/commandInterpreter/showConfirmationDialogClientCommandHandler.ts
import { ClientCommandHandlerBase } from 'tessa/workflow/krProcess/clientCommandInterpreter';
import { IClientCommandHandlerContext } from 'tessa/workflow/krProcess';
import { getTypedFieldValue } from 'tessa/platform';
import { showMessage } from 'tessa/ui';

export class ShowConfirmationDialogClientCommandHandler extends ClientCommandHandlerBase {

  public async handle(context: IClientCommandHandlerContext) {
    const text = getTypedFieldValue(context.command.parameters['text']);

    if (text) {
      await showMessage(text);
    }
  }

}

File: Source/Extensions/Tessa.Extensions.Default.Client/Workflow/KrProcess/CommandInterpreter/ShowConfirmationDialogClientCommandHandler.cs
using System.Threading.Tasks;
using Tessa.Extensions.Default.Shared.Workflow.KrProcess.ClientCommandInterpreter;
using Tessa.Platform.Validation;
using Tessa.UI;

namespace Tessa.Extensions.Default.Client.Workflow.KrProcess.CommandInterpreter
{
    /// <summary>
    /// Обработчик клиентской команды <see cref="DefaultCommandTypes.ShowConfirmationDialog"/>.
    /// </summary>
    public sealed class ShowConfirmationDialogClientCommandHandler: ClientCommandHandlerBase
    {
        #region Base Overrides

        /// <inheritdoc/>
        public override async Task Handle(
            IClientCommandHandlerContext context)
        {
            var command = context.Command;
            if (command.Parameters.TryGetValue("text", out var textObj)
                && textObj is string text
                && !string.IsNullOrWhiteSpace(text))
            {
                await TessaDialog.ShowNotEmptyAsync(ValidationResult.FromText(text));
            }
        }

        #endregion
    }
}